testsuite/libra-fuzzer/src/fuzz_targets/secure_json_rpc_client.rs

```diff
@@ -19,12 +27,14 @@ impl FuzzTargetImpl for SecureJsonRpcSubmitTransaction {
         "Secure JSON RPC submit_transaction() response"
     }
 
-    fn generate(&self, _idx: usize, _gen: &mut ValueGenerator) -> Option<Vec<u8>> {
```

Comment:
you'll need a generate function for all of these, check my previous PR on how to write that : o